### PR TITLE
Create GetMeetingParticipantReports method

### PR DESCRIPTION
**Pull request by steelbrain**
As advertised

Also moves common stuff out of individual factories

### CODE REVIEW

File: src/reports.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { PaginatedResponse } from './common';
+import request from './util/request';
+
+export type GetMeetingParticipantReportsParams = {
+  page_size?: number
+  next_page_token?: string
+  include_fields?: 'registrant_id'
+};
+
+export interface GetMeetingParticipantReportsResponse extends PaginatedResponse {
```
Comment (sedenardi) [inline src/reports.ts]: Please change this to a `type` to match the existing convention.

**Comment by sedenardi** [comment]
This PR looks good, I'll merge once the one comment is resolved.

As an aside, it looks like Zoom introduce a new `next_page_token` parameter for pagination, and [seems to be](https://marketplace.zoom.us/docs/api-reference/pagination) recommending using it going forward. Future versions of this library (breaking versions), will use that parameter.

File: src/reports.ts (hunk 1 above)
Comment (steelbrain) [reply src/reports.ts]: Done!

**Comment by steelbrain** [comment]
Thanks for the note about `next_page_token`, not sure if you noticed but that's what the PR is using!

Thanks again for the feedback, what do you think of the new formatting?

**Comment by sedenardi** [comment]
> Thanks for the note about `next_page_token`, not sure if you noticed but that's what the PR is using!
> 
> Thanks again for the feedback, what do you think of the new formatting?

Everything looks good!

As I noted, future version will probably replace `PaginatedResponse` with something like `PageTokenResponse`, and requests that support paging via the `next_page_token` will union with a `PageTokenRequest` type. But for now everything looks good.

Will merge shortly.

**Comment by sedenardi** [comment]
@steelbrain Published to `npm` as version `1.5.0`. Thanks again for your contribution!

**Pull request merged**